pybaseball/playerid_lookup.py

```diff
@@ -60,21 +60,19 @@ def get_lookup_table(save=False):
 
 def name_similarity(last: str, first: str, player_table: pd.DataFrame) -> pd.DataFrame:
     """Calculates similarity of first and last name provided with all players in player_table
-
     Args:
         last (str): Provided last name
         first (str): Provided first name
         player_table (pd.DataFrame): Chadwick player table including names
-
     Returns:
         pd.DataFrame: 5 nearest matches from fuzzywuzzy.process
     """
     filled_df = player_table.fillna("")
     chadwick_names = filled_df["name_first"] + " " + filled_df["name_last"]
     fuzzy_matches = pd.DataFrame(process.extract(f"{first} {last}", chadwick_names, limit=5))
-    matched_names = fuzzy_matches[0].str.split(expand=True)
-    matched_names = matched_names.rename(columns={0: "name_first", 1: "name_last"})
-    return matched_names
+    fuzzy_indices = fuzzy_matches[2].tolist()
```

Comment:
same comment as below, instead of returning the indexes, how about subsetting the dataframe and returning that?
i.e.
```
return filled_df.iloc[fuzzy_indices].reset_index(drop=True)
```

Reply:
done